From 62d1333ba476a382e2047b6cc9eb2f601203da1a Mon Sep 17 00:00:00 2001
From: Vipul Kumar <vipul_kumar@mentor.com>
Date: Mon, 18 Mar 2019 17:22:18 +0530
Subject: [PATCH 1/2] arm: dts: imx6ul: Adding missing clock source

commit  1db813140b8cb401db1f67c969dad0e67d906589 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch added missing clock source under cpu node. Without these
sources, getting following error at boot time:

cpu cpu0: Failed to get clk 'pll1_bypass': -2
imx6q-cpufreq: probe of imx6q-cpufreq failed with error -2

Signed-off-by: Vipul Kumar <vipul_kumar@mentor.com>
Signed-off-by: Xiaolei Wang <Xiaolei.Wang@windriver.com>
---
 arch/arm/boot/dts/imx6ul.dtsi | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 7bf3ce880fec..6105a17f3039 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -83,10 +83,15 @@
 				 <&clks IMX6UL_CA7_SECONDARY_SEL>,
 				 <&clks IMX6UL_CLK_STEP>,
 				 <&clks IMX6UL_CLK_PLL1_SW>,
-				 <&clks IMX6UL_CLK_PLL1_SYS>;
+				 <&clks IMX6UL_CLK_PLL1_SYS>,
+				 <&clks IMX6UL_PLL1_BYPASS>,
+				 <&clks IMX6UL_CLK_PLL1>,
+				 <&clks IMX6UL_PLL1_BYPASS_SRC>,
+				 <&clks IMX6UL_CLK_OSC>;
 			clock-names = "arm", "pll2_bus",  "pll2_pfd2_396m",
 				      "secondary_sel", "step", "pll1_sw",
-				      "pll1_sys";
+				      "pll1_sys", "pll1_bypass", "pll1",
+				      "pll1_bypass_src", "osc";
 			arm-supply = <&reg_arm>;
 			soc-supply = <&reg_soc>;
 		};
-- 
2.17.1

